examples/event_watcher.rs

```diff
@@ -1,4 +1,6 @@
-use futures::{pin_mut, TryStreamExt};
+use std::pin::pin;
+
+use futures::TryStreamExt;
```

Comment:
in this case this seems like a worse api for users; chaining watcher apis become harder to read, and people need `futures` anyway

Reply:
I would not think it is particularly difficult to read, but API style is largely a matter of preference, so if it is not your preference, I will revert to `pin_mut`.

In a similar style to `pin_mut`, it can be written as follows, how about this?

```rust
let event_stream = watcher(events, conf).default_backoff().applied_objects();
let mut event_stream = pin!(event_stream);
```

Reply:
ah, yeah, that second line pin makes more sense for at least the user-facing part in the examples. the internals in the runtime look fine to me.

Reply:
Moved pin value in examples to independent sentence.